src/containers/flows/AcceptBid/AcceptBid.constants.tsx

```diff
@@ -0,0 +1,55 @@
+//- Types
+import { Step } from './AcceptBid.types';
+
+export const TITLES = {
+	[Step.Details]: { PRIMARY: 'Accept Bid' },
+	[Step.CheckingZAuctionApproval]: { PRIMARY: 'zAuction Approval' },
+	[Step.Confirmation]: { PRIMARY: 'Are you sure?' },
+	[Step.Success]: { PRIMARY: 'Bid Accepted' },
+};
+
+export const BUTTONS = {
+	[Step.Details]: { PRIMARY: 'Accept', SECONDARY: 'Cancel', TERTIARY: 'Retry' },
+	[Step.Confirmation]: {
+		PRIMARY: 'Confirm',
+		SECONDARY: 'Cancel',
+		TERTIARY: 'Retry',
+	},
+	[Step.Success]: { PRIMARY: 'Finish' },
+};
+
+export const STATUS_TEXT = {
+	CHECK_ZAUCTION: 'Checking status of zAuction approval...',
+	ACCEPT_ZAUCTION_PROMPT:
+		'Before you can accept a bid, your wallet needs to approve zAuction. You will only need to do this once. This will cost gas.',
+	APPROVING_ZAUCTION:
+		'Approving zAuction. This may take up to 20 mins... Please do not close this window or refresh the page.',
+	AWAITING_APPROVAL: 'Waiting for approval from your wallet...',
+	APPROVED: 'Approved',
+};
+
+export const ERRORS = {
+	SIGNATURE: 'Failed to generate signature.',
+	TRANSACTION: 'Transaction failed.',
+	LIBRARY: 'Failed to connect with Web3 wallet.',
+	CONSOLE_TEXT: 'Failed to check zAuction approval status',
+	REJECTED_WALLET: 'Rejected by wallet',
+};
+
+export const MESSAGES = {
+	TEXT_LOADING: 'Loading Bid Data...',
+	TEXT_FAILED_TO_LOAD: 'Failed to load bid data.',
+	TEXT_CONFIRMATION:
+		'This transaction is about to be seared upon the blockchain. There’s no going back.',
+	TEXT_ACCEPT_PROMPT: 'Please accept wallet transaction...',
+	TEXT_ACCEPTING_BID: 'Accepting bid. Please wait - transaction is processing.',
+	TEXT_CONFIRM_ACCEPT:
+		"Are you sure you want to accept this bid? You can't undo this.",
+	TEXT_WAITING_FOR_WALLET:
+		'Waiting for transaction approval. You should receive a request in your wallet.',
+	SUCCESS_CONFIRMATION: 'Success! Bid accepted and ownership transferred',
+	CONFIRM_BID_AMOUNT: 'Are you sure you want to accept a bid of',
+};
+
+export const getNFTConfirmDetailsText = (domainName: string) =>
+	`and transfer ownership of ${domainName} to `;
```

Comment:
I think previously the domain names in these flows are bold, so you could make this function return a `ReactNode`, i.e. `=> <>and transfer ownership of <b>{domainName}</b> to</>`

Reply:
tidied this up a bit more as Eth requested some changes too :)